geocoding/location-service/pyproject.toml

```diff
@@ -46,6 +46,7 @@ uritemplate = "3.0.1"
 urllib3 = "1.26.4"
 Werkzeug = "1.0.1"
 WTForms = "2.3.3"
+ratelimiter = "^1.2.0"
```

Comment:
This will update ratelimiter to the latest version on `poetry update`, should probably be `= "1.2.0"`

Reply:
Good catch, thanks